### ISSUE TITLE
Any way to select predefined filters that already exist?

### ISSUE THREAD

**Issue by stacksjb** [selfire1/todoist-focus-mode#4]
Hello,

I found this and immediately fell in love - I would love to use it.

However, I spent hours trying to figure out how to use it... until I figured out you have to scroll down and add the filters.

Is there no way to select existing filters? I have to recreate them all manually?

**Comment by stacksjb** [comment]
If I could just select filters from my existing filter list, this would be much easier to use for my workflow, as I have 12 filters I work through throughout the day (switching among them)

**Comment by selfire1** [comment]
Hi! Thanks for providing the feedback. This is a popular feature request and I'm looking into how to support it!

**Comment by stacksjb** [comment]
FYI, there isn't a way to call Todoist's API and return tasks matching a known filter. This would require two calls - one to get the stored/predefined filters, then another to query for tasks matching the filter that you got from that call.

**Comment by selfire1** [comment]
Yeah, you're right. I'm thinking to get the user's filters when they first open the website, and then present a dropdown with stored filters.
